templates/dashboard/includes/_menu_items.html

```diff
@@ -32,9 +32,12 @@
           {% endif %}
         </tr>
       </thead>
-      <tbody id="menu-items" class="menu-items" data-post-url="{% if menu_item %}{% url 'dashboard:menu-items-reorder' menu_pk=menu.pk root_pk=menu_item.pk %}{% else %}{% url 'dashboard:menu-items-reorder' menu_pk=menu.pk %}{% endif %}">
+      <tbody class="sortable-items-container"
+             data-post-parameter="ordered_menu_items"
+             data-post-url="{% if menu_item %}{% url 'dashboard:menu-items-reorder' menu_pk=menu.pk root_pk=menu_item.pk %}{% else %}{% url 'dashboard:menu-items-reorder' menu_pk=menu.pk %}{% endif %}"
+      >
```

Comment:
I'm not sure about Saleor's code style. Do I have to return `>` to a new line or keep it on the same line?

Reply:
Soon™: https://github.com/prettier/prettier/issues/1882